e2etest/newe2e_resource_manager_azstorage.go

```diff
@@ -20,7 +22,7 @@ type AzureAccountResourceManager struct {
 	armClient *ARMStorageAccount
 }
 
-func (acct *AzureAccountResourceManager) ApplySAS(URI string, loc common.Location, optList ...GetURIOptions) string {
+func (acct *AzureAccountResourceManager) ApplySAS(URI string, loc common.Location, lev cmd.LocationLevel, et common.EntityType, optList ...GetURIOptions) string {
```

Comment:
entity type seems unused? Were these intended to be for a check of some sort? E.g. that using a service sas doesn't make sense on an account?

Reply:
Entity is mostly intended for datalake + file share and only to specify file vs directory SAS. Since its not applicable for container/account levels, we set entity to the empty value.